cf/command_factory/factory.go

```diff
@@ -124,6 +124,7 @@ func NewFactory(ui terminal.UI, config core_config.ReadWriter, manifestRepo mani
 	factory.cmdsByName["logs"] = application.NewLogs(ui, config, repoLocator.GetLogsRepository())
 	factory.cmdsByName["oauth-token"] = commands.NewOAuthToken(ui, config, repoLocator.GetAuthenticationRepository())
 	factory.cmdsByName["org"] = organization.NewShowOrg(ui, config)
+	factory.cmdsByName["user-info"] = user.ShowUserInfo(ui, config, repoLocator.GetUserRepository())
```

Comment:
I think better to put "user-info" command registration line after 'update-security-group' command registration as most of commands are in alphabetical order.